app/models/krikri/activity.rb

```diff
@@ -83,5 +83,23 @@ def rdf_subject
       RDF::URI(Krikri::Settings['marmotta']['ldp']) /
         Krikri::Settings['prov']['activity'] / id.to_s
     end
+
+    ##
+    # Return an Enumerator of URI strings of entities (e.g. aggregations or
+    # original records) that pertain to this activity
+    #
+    # @return [Enumerator] URI strings
+    def generated_entity_uris
+      activity_uri = RDF::URI(rdf_subject)  # This activity's LDP URI
+      query = Krikri::ProvenanceQueryClient.find_by_activity(activity_uri)
+      query.each_solution.lazy.map do |s|
+        s.record.to_s
```

Comment:
`Krikri::OriginalRecord#initialize` wants to work with a string, or something that has `#to_s`.  I must have figured I'd cast it as a string if that's the way it was going to be used.   `ActiveTriples::Resource#set_subject!` takes `uri_or_str` too, so think that I decided to cast it as a string at this point to make sure it _can_ be cast as a string, rather than letting any error with that result in a longer stacktrace.

I don't have anything against letting it be an `RDF::URI`, it that would be useful in the future, but right now all it's being used for is instantiating Aggregations or OriginalRecords.